changelog/add-update-email-type-key

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: changed
+
+Added underscore prefix in sensei_email_type key
```

Comment:
I don't think this PR needs a changelog entry as it's part of a new feature.

Reply:
Right, thanks, removed it here https://github.com/Automattic/sensei/pull/6544/commits/8b73bd6af83bdb718f2824c51e2e22d124ae992d